### ISSUE TITLE
Faster R-CNN with Res50 backbone

### ISSUE THREAD

**Issue by a-maci** [pytorch/vision#1158]
Is there a trainer script for faster rcnn (like there is for mask-rcnn in references/detection/)?
Will it be quite different from mrcnn or is it just a matter of importing the model and specifying --model to be fastrcnn?

Also, what is the command line to train the frcnn model from scratch to get 37.0% AP?

**Comment by fmassa** [comment]
the training script in `references/detection` works for all detection-based models, i.e., Faster R-CNN, Mask R-CNN and Keypoint R-CNN.

Here is the command I used
```
python -m torch.distributed.launch --nproc_per_node=8 --use_env train.py --dataset coco --model fasterrcnn_resnet50_fpn --epochs 26 --lr-steps 16 22 --aspect-ratio-group-factor 3
```
